tests/robustness/main_test.go

```diff
@@ -96,8 +96,11 @@ func testRobustness(ctx context.Context, t *testing.T, lg *zap.Logger, s testSce
 		t.Fatal(err)
 	}
 
-	watchProgressNotifyEnabled := c.Cfg.ServerConfig.ExperimentalWatchProgressNotifyInterval != 0
-	validateGotAtLeastOneProgressNotify(t, r.Client, s.watch.requestProgress || watchProgressNotifyEnabled)
+	failpointImpactingWatch := s.failpoint == failpoint.SleepBeforeSendWatchResponse
```

Comment:
this seems to be a important failpoint to test progress notify. Is there plan to investigate further?

Reply:
No, with progress notifies not being guaranteed in every situation, it's hard to make a strong requirement on them when we use failpoints that directly disrupt their delivery. We can consider revisiting it only after we improve etcd progress guarantees.